### ISSUE TITLE
Problems when running example

### ISSUE THREAD

**Issue by AlexPars** [kl-lab/dejavu#2]
I am running the example using Reference Data from M3 competition, and getting some errors

I was able to solve first error, but stuck at next error

First error was when running dejavu::Similarity function in the example
fc_Simi <- Similarity(AirPassengers, fh = 20, LoadData = TRUE, path = NULL)

  bad restore file magic number (file may be corrupted) -- no data loaded
In addition: Warning message:
file ‘ref12.RData’ has magic number 'versi'
  Use of save versions prior to 2 is deprecated 

I have solved this error by reading the RData files using read.csv, and then saving the output of read.csv using R function save

Second error was when running Similarity with the newly saved RData files
fc_Simi <- Similarity(AirPassengers, fh = 20, LoadData = TRUE, path = NULL)

Error in Forecasting(y.train, fh = fh, nts = nts, Preprocessing = Preprocessing,  : 
  object 'ref_info' not found

**Comment by feng-li** [comment]
Did you use the most recent version of https://github.com/kl-lab/dejavu-refdata ?

**Comment by AlexPars** [comment]
yes, I took the latest version of  https://github.com/kl-lab/dejavu-refdata

**Comment by feng-li** [comment]
We have tested with R version 4.0.4 and everything is fine. Please make sure that you have cloned the reference data properly. Note that the reference git repository is git-lfs controlled. Using GitHub Download ZIP option will only download the meta data and the binary will not be downloaded.

Or you could use the releases page link to download the reference binary files.
